engines/python/setup/djl_python_engine.py

```diff
@@ -117,6 +117,7 @@ def run_server(self):
                     outputs = Output(code=204, message="No content")
             except Exception as e:
                 logging.exception("Failed invoke service.invoke_handler()")
+                logging.exception(e)
```

Comment:
logging.exception suppose print out the full stack trace, did you observe more information in model exception unit test?